scripts/ci/generate_install_lists.py

```diff
@@ -74,6 +74,11 @@ def main(argv=sys.argv[1:]):
                 package_root = all_packages[package]
                 Path(package_root, 'COLCON_IGNORE').touch()
 
+        # generate a list of non-ROS package directories
```

Comment:
Would it make more sense to move this code close to where it is being used, just before line 108?

Reply:
Eh, sure: e3737f8